src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.model;
+
+public final class Fruit {

```

Comment:
if Fruit will contain also its amount, logic with collections can be simpler. but it's rather a wish.

Reply:
I agree, but this option is ok too